lib/datasets/california-housing.rb

```diff
@@ -0,0 +1,70 @@
+require "csv"
+require "zip"
+
+require_relative "dataset"
+require "rubygems/package"
+
+module Datasets
+  class CaliforniaHousing < Dataset
+    Record = Struct.new(:medianHouseValue,
```

Comment:
Could you use `snake_case` instead of `camelCase`?

Reply:
Done.